woocommerce/woocommerce-android
# [TTP] Add badges to highlight TTP on more menu button and menu screen

Closes: #8922


### Description

Adds badges to attract more attention to the newly added TTP feature to the "menu" button and payments button on the menu screen

I also renamed `IsTapToPayAvailability` -> `TapToPayAvailabilityStatus`

### Testing instructions

Use clean build
* Notice the red badge on the menu button
* Open the menu screen and notice the badge on the row of the payment

The red badge on the menu should be gone after you see the row of the payment once. The badge on the row of the payment should go after it's clicked once.

Badges should not be visible if TTP is not available, because the device is not supported, the country or feature is not enabled via beta flags menu

### Images/gif


https://user-images.githubusercontent.com/4923871/235657866-9887a27a-b81d-4def-a202-ab43daecab2e.mp4
